lua/nvim-web-devicons.lua

```diff
@@ -1492,13 +1492,13 @@ local icons_by_file_extension = {
   ["sol"] = {
     icon = "ﲹ",
     color = "#519aba",
-    cterm_color = "67",
+    cterm_color = "74",
     name = "Solidity",
   },
   ["prisma"] = {
     icon = "卑",
     color = "#ffffff",
-    cterm_color = "white",
+    cterm_color = "231",
```

Comment:
big win

Reply:
It's the same color.

The script only checks for numbers actually, not for color names. Originally I used a macro, that plugin also works with color names. Maybe I should update the script so that it also catches color names.

Reply:
What would you do instead?

Reply:
It's correct... I'm just happy we are using colours consistently now ;)

Reply:
But it doesn't really matter, as long as gui colors are written in html format, even a zero will be turned into the right color. It's not a big issue.

Reply:
That's ok then 👍🏻